src/Microsoft.AspNet.SignalR.SqlServer/TaskAsyncHelper.cs

```diff
@@ -0,0 +1,1194 @@
+// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
```

Comment:
It is used, I thought to change to use shared class from Microsoft.AspNet.SignalR.Common after refactor.